barretenberg/cpp/src/barretenberg/sumcheck/sumcheck_round.hpp

```diff
@@ -263,6 +300,32 @@ template <typename Flavor> class SumcheckProverRound {
         Utils::apply_to_tuple_of_tuples(tuple, extend_and_sum);
     }
 
+    /**
+     * @brief Compute Libra round univariate expressed given by the formula
+    \f{align}{
+        \texttt{libra_round_univariate}_i(k) =
+        \rho \cdot 2^{d-1-i} \left(\sum_{j = 0}^{i-1} g_j(u_{j}) + g_{i,k}+
+        \sum_{j=i+1}^{d-1}\left(g_{j,0}+g_{j,1}\right)\right)
+        =  \texttt{libra_univariates}_{i}(k) + \texttt{libra_running_sum}
+    \f}.
+     *
+     * @param zk_sumcheck_data
+     * @param round_idx
+     */
+    static SumcheckRoundUnivariate compute_libra_round_univariate(ZKSumcheckData<Flavor> zk_sumcheck_data,
```

Comment:
do you think it would be more readable? cause sumcheck prover round is operating on the size of polynomials/2^{round_idx}. i thought about deriving it this way, but seems more complicated